modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
 
 toc::[]
 
+== Infinite Scale 5.0.0
+[discrete]
+=== General
+
+This release brings significant enhancements in performance and stability, particularly for deployments based on Kubernetes. It is also designed to provide an optimized experience for both end users and administrators.
+Please find the most important changes here described or refer to the changelog for a complete list of changes:
+
+* Infinite Scale: https://github.com/owncloud/ocis/releases/tag/v5.0.0[Changes in 5.0.0]
+* Web: https://github.com/owncloud/web/releases/tag/v8.0.0[Changelog for ownCloud Web 7.1.0]
+
+[discrete]
+=== Federated Cloud Sharing
```

Comment:
This is missing a "feature preview" disclaimer. This is far from production ready. Hard veto for making it look like it's ready.

Reply:
oh yes, had the disclaimer in the pptx, missed it here. thanks, important catch!